src/BudgetManager/BudgetManager.API/Seeding/Seeds/users.json

```diff
@@ -1,32 +1,37 @@
 [
   {
-    "Id": "a0ffbfb1-fbd1-4472-bc18-324f9bd41e5d",
     "fullName": "Panther Seed",
     "dob": "2022-07-28T15:19:42.251Z",
     "email": "seed1@seed.com"
   },
   {
-    "Id": "a1ffbfb1-fbd1-4472-bc18-324f9bd41e5d",
     "fullName": "Joker Seed",
     "dob": "2022-07-28T15:19:42.251Z",
     "email": "seed2@seed.com"
   },
   {
-    "Id": "a2ffbfb1-fbd1-4472-bc18-324f9bd41e5d",
     "fullName": "Morgana Seed",
     "dob": "2022-07-28T15:19:42.251Z",
     "email": "seed3@seed.com"
   },
   {
-    "Id": "a3ffbfb1-fbd1-4472-bc18-324f9bd41e5d",
     "fullName": "Lelouch Seed",
     "dob": "2022-07-28T15:19:42.251Z",
     "email": "seed4@seed.com"
   },
   {
-    "Id": "a4ffbfb1-fbd1-4472-bc18-324f9bd41e5d",
     "fullName": "Nagisa Seed",
     "dob": "2022-07-28T15:19:42.251Z",
     "email": "seed5@seed.com"
+  },
+  {
+    "fullName": "Olesya Vasilieva",
+    "dob": "2022-07-28T15:19:42.251Z",
+    "email": "OlesyaVasilieva@gmail.com"
+  },
+  {
+    "fullName": "Nikita Bobrov",
+    "dob": "2022-07-28T15:19:42.251Z",
```

Comment:
dob must be changed to DateOfBirth, otherwise, it throws Aggregation exception again. Of course, in order to do that, the DB must be dropped, and my PR must be merged.